strings/nuclear_disk_quips.json

```
@@ -0,0 +1,188 @@
+		"Uhm... nice posters...",
+		"Y'know, I'm not sure I like you. I'm not sure I like you at all.",
+		"Y'think TerraGov cares about these warheads?",
+		"YOU'RE GOING TO KILL ME, AREN'T YOU?!"
+	],
+	"On Unfavorable Shuttle":
+	[
+		"Are you SURE you're in the right place?",
+		"BE FUCKING CAREFUL WITH THAT CONSOLE!",
+		"Hey, be careful about sending this shuttle.",
+		"Hey, I don't think you should send this shuttle. At least leave me behind.",
+		"I hear the weather's shit where we're going.",
+		"I looked at the destination board and it's not looking good for us.",
+		"I'm a clingy bastard who likes the station, and this isn't looking good at ALL.",
+		"I'm not pissed off now, but I will be very soon",
+		"My code is sternly rejecting spending another second on this shuttle.",
+		"Ruh-roh, this isn't going to a good spot for me.",
+		"Wait, I don't like where this shuttle is going.",
+		"WAIT, WHAT THE FUCK ARE YOU DOING ON THIS SHUTTLE?!",
+		"WOAH, don't wanna send me there, haha.",
+		"You know, maybe this destination selector is lying. Maybe you know what you're doing. But do you really?"
+	],
+	"On Station":
+	[
+		"Ah, good times in a good place.",
+		"Have you noticed just how dirty this station is? Yucky floor. Any station floor is good floor though.",
+		"Honestly, the only thing I care about these days is just being inside.",
+		"How's your mood doing? I'm doing alright.",
+		"I don't speak often when it's not needed, but I think I needed to break up whatever monotony was going on.",
+		"I like being safe here!",
+		"I'm glad you're not a monster.",
+		"I'm the real deal, you know?",
+		"If you were a bad guy, would you tell me?",
+		"If you work for Nanotrasen, you're doing them proud. If otherwise... well, I'm not sure what to say.",
+		"If you're feeling hungry, I think I saw the chef cook something nice up earlier. Vending machines are also a good bet.",
+		"If you're wondering if I like you, the answer is dependent on us staying roughly where we are.",
+		"Much rather in than out! Haha!",
+		"You're an interesting fellow, you know that?"
+	],
+	"Pissing Off":
+	[
+		"BRAVO!!! BRAVO!!!!!!",
+		"Dentskull.",
+		"Did you fuck up on purpose?",
+		"F for Effort.",
+		"Fuck you, I'm out of here. Big ol' middle finger.",
+		"Go eat an Arnold Pizza.",
+		"Here's hoping I land right on a bed!",
+		"I don't think you could have screwed that up harder if you actually TRIED.",
+		"I don't want to be terribly abusive, but I gotta prioritize my own needs.",
+		"I HAVE NEEDS!!! AND YOU DON'T RESPECT THEM!!!",
+		"I hope you-GLORF!",
+		"I might have words with my people over this.",
+		"I think you knew what you were doing.",
+		"I'm going to have rats chew on my plastic in some shitty maintenance tunnel, and it is All. Your. Fault.",
+		"If I had a slow-clap processor, it'd be going right now.",
+		"If I warned you, don't say I didn't!",
+		"Maybe a drink will calm me down?",
+		"No lie: that was a bad choice.",
```

```suggestion
		"No lie: That was a bad choice.",
```
not sure on this one